packages/cursorless-engine/src/core/updateSelections/RangeUpdater.ts

```diff
@@ -96,6 +96,10 @@ export class RangeUpdater {
   private listenForDocumentChanges() {
     this.disposable = ide().onDidChangeTextDocument(
       (event: TextDocumentChangeEvent) => {
+        const changes = event.contentChanges;
+        console.debug(
+          `RangeUpdater.listenForDocumentChanges(): changes=${JSON.stringify(changes)}`,
+        );
```

Comment:
huh yeah I think we agreed we should be able to support better logging. i'll need to verify that this doesn't result in a bunch of spam when we ship. actually I think we have a debug output mechanism which you could use. search our code for debug. I believe it's guarded by a setting that's off by default in prod